tools/defaults.bzl

```diff
@@ -213,6 +199,8 @@ def ng_web_test_suite(deps = [], static_css = [], exclude_init_script = False, *
         "//src/material/prebuilt-themes:azure-blue",
     ]
 
+    bootstrap = []
```

Comment:
we don't ever put anything in this bootstrap list, can we just remove it?

Reply:
The code below adds something for CSS, so it's used